src/utils/env.ts

```diff
@@ -86,3 +86,11 @@ export const getReferralHashtags = (): string => {
     assert(referralHashtags, 'NEXT_PUBLIC_REFERRAL_HASHTAGS is not set');
     return referralHashtags;
 };
+
+export const getSubgraphUrl = (chainId: number): string | undefined => {
+    if (chainId === 314) {
+        return process.env.NEXT_PUBLIC_SUBGRAPH_URL_314;
```

Comment:
it does not assert the case when this env variable is missing.

Reply:
That's intentional. In the dev or stg environments, this variable is not set. So `undefined` should be returned when this env variable is missing.